x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/rollup/ConfigTestHelpers.java

```diff
@@ -125,6 +96,39 @@ public static String getCronString() {
                 " " + (ESTestCase.randomBoolean() ? "*" : String.valueOf(ESTestCase.randomIntBetween(1970, 2199)));  //year
     }
 
+    public static List<MetricConfig> randomMetricsConfigs(final Random random) {
```

Comment:
Mostly just curious, but why Random as an argument instead of using the test class' random methods (`randomBoolean()`, etc)?

Reply:
I wasn't sure that ConfigTestHelpers methods were always used in the context of an ESTestCase, and I expect to have to use these methods in the high level rest client too so I found it safer to pass the Random object to the methods similarly to what is done for RandomNumbers or RandomPicks.

Reply:
:+1: cool, thanks for the explanation :)